src/style.css

```diff
@@ -0,0 +1,132 @@
+* {
+  box-sizing: border-box;
+}
+
+h2,
+p,
+body {
+  margin: 0;
+}
+
+:root {
+  --dark-text: #060b35;
+  --light-text: #616070;
+  --button-color: #00acdc;
+}
+
+.page {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  align-self: center;
+  width: 160px;
+  height: 134px;
+  object-fit: contain;
+  margin: 0 0 40px;
+}
+
+.card__title {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: var(--dark-text);
+  margin: 0 0 4px;
+}
+
+.card__code {
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  color: var(--light-text);
+  margin: 0 0 16px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 0 24px;
+}
+
+.card__reviews {
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  color: var(--dark-text);
+}
+
+.card__price-section {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 0 16px;
+}
+
+.card__price-text {
+  font-size: 12px;
+  line-height: 18px;
```

Comment:
Try to use `line-height` as a relative value or as a percentage instead of `px`.
Example:
```
line-height: 1.5;
```
It will be calculated from font-size automatically (12px * 1.5 = 18px for your case)

Reply:
Thank you for your guidance.